app/src/main/java/org/wikipedia/diff/ArticleEditDetailsFragment.kt

```diff
@@ -346,6 +346,7 @@ class ArticleEditDetailsFragment : Fragment(), WatchlistExpiryDialog.Callback, L
         }
 
         binding.watchButton.setOnClickListener {
+            PatrollerExperienceEvent.logAction(if (isWatched) "unwatch_init" else "watch_init", "pt_toolbar")
```

Comment:
Does this need to be only in Patroller Tasks by using `sendPatrollerExperienceEvent`?